docs/json_rpc_api.mediawiki

```diff
@@ -2177,8 +2177,6 @@ of the best block.
 |
 <code>(json object)</code>
 : <code>data</code>: <code>(string)</code> hex-encoded block data
-: <code>hash1</code>: <code>(string)</code> (DEPRECATED) hex-encoded formatted hash buffer
-: <code>midstate</code>: <code>(string)</code> (DEPRECATED) hex-encoded precomputed hash state after hashing first half of the data
 : <code>target</code>: <code>(string)</code> the hex-encoded little-endian hash target
 
 <code>{"data": "hex", "hash1": "hex", "midstate": "hex", "target": "hex"}</code>
```

Comment:
This example return should be updated as well.

Reply:
Good call.  Updated.